docs/deployment/getting-started-locally/getting-started-locally.md

```diff
@@ -11,27 +11,27 @@ Before we can setup `etcd-druid` and use it to provision `Etcd` clusters, we nee
 `etcd-druid` uses [kind](https://kind.sigs.k8s.io/) as it's local Kubernetes engine. The local setup is configured for kind due to its convenience only. Any other Kubernetes setup would also work. 
 
 ```bash
-> make kind-up
+make kind-up
 ```
 
 This command sets up a new Kind cluster and stores the kubeconfig at `./hack/kind/kubeconfig`.  Additionally, this command also deploys a local container registry as a docker container. This ensures faster image push/pull times. The local registry can be accessed as `localhost:5001` for pushing and pulling images. 
 
 To target this newly created cluster, set the `KUBECONFIG` environment variable to the kubeconfig file.
 
 ```bash
-> export KUBECONFIG=$PWD/hack/kind/kubeconfig
+export KUBECONFIG=$PWD/hack/kind/kubeconfig
 ```
 
 > **Note:**  If you wish to configure kind cluster differently then you can directly invoke the script and check its help to know about all configuration options.
->
-> ```bash
-> > ./hack/kind-up.sh -h
->   usage: kind-up.sh [Options]
->   Options:
->     --cluster-name  <cluster-name>   Name of the kind cluster to create. Default value is 'etcd-druid-e2e'
->     --skip-registry                  Skip creating a local docker registry. Default value is false.
->     --feature-gates <feature-gates>  Comma separated list of feature gates to enable on the cluster.
-> ```
+
+```bash
+./hack/kind-up.sh -h
+  usage: kind-up.sh [Options]
+  Options:
+    --cluster-name  <cluster-name>   Name of the kind cluster to create. Default value is 'etcd-druid-e2e'
+    --skip-registry                  Skip creating a local docker registry. Default value is false.
+    --feature-gates <feature-gates>  Comma separated list of feature gates to enable on the cluster.
+```
 
 ## 02-Setting up etcd-druid
```

Comment:
Can you remove the prepending numbers for each section in the entire file? It is not consistent with the rest of the documentation.

```suggestion
## Setting up etcd-druid
```

Reply:
I prefer having numbers. This document provide a sequence of steps as compared to other documents.